core/lib/resources/presence/presence_manager.js

```diff
@@ -39,13 +40,30 @@ PresenceManager.prototype.setup = function() {
   });
 
   // save so you removeListener on destroy
-  this.sentryListener = function(sentry) {
+  this.sentryListener = function (sentry) {
+    if (manager.destroying) return;
```

Comment:
The flag name is fine. I am just pointing out that this check is never going to be true.